src/main/webapp/styles/main.css

```diff
@@ -105,14 +106,46 @@ p {
 
 input[type=text], select {
     width: 100%;
-    padding: 12px 20px;
+    padding: 12px 10px;
     margin: 8px 0;
     display: inline-block;
     border: 1px solid #ccc;
     border-radius: 4px;
     box-sizing: border-box;
 }
 
+#blueButton{
```

Comment:
Could we please rename the `blueButton` and `pinkButton` ID to something like `blueAppointmentButton` and `pinkAppointmentButton`? My IDE is giving me warnings because this ID has the same name as the original class.